tasks/test.py

```diff
@@ -592,7 +592,7 @@ def install_shellcheck(ctx, version="0.7.0", destination="/usr/local/bin"):
         platform = "linux"
 
     ctx.run(
-        "wget -qO- \"https://storage.googleapis.com/shellcheck/shellcheck-v{sc_version}.{platform}.x86_64.tar.xz\" | tar -xJv -C /tmp".format(
+        "wget -qO- \"https://github.com/koalaman/shellcheck/releases/download/v{sc_version}/shellcheck-v{sc_version}.{platform}.x86_64.tar.xz\" | tar -xJv -C /tmp".format(
```

Comment:
nit: we might want to do this in a separate PR, just for a cleaner history (but feel free to ignore).

Reply:
It's already on master, I just cherrypicked because CircleCI was failing without this URL 😅